python/pyplugin_installer/installer.py

```diff
@@ -649,6 +649,7 @@ def installFromZipFile(self, filePath):
                     keepTrying = False
 
         if success:
+            QApplication.setOverrideCursor(Qt.WaitCursor)
```

Comment:
```
from qgis.utils import OverrideCursor

with OverrideCursor(Qt.WaitCursor):
     do_someting_that_could_except()
```

should be a bit more solid here

Reply:
Nice suggestion, let's apply it in the whole file then.